### PR TITLE
fix local storage

### PR DESCRIPTION
**Pull request by tarcisiozf**
when this lib is imported by electron fs gets replaced by an empty mock, we should lazy load those functions

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+nodejs 12.22.8
```
Comment (vigan-abd) [inline .tool-versions:1]: what's .tool-versions, is it necessary to be included in repo?

Comment (tarcisiozf) [reply .tool-versions:1]: It's a common pattern for IDEs and version managers (like asdf) to detect the best version to use

Comment (tarcisiozf) [reply .tool-versions:1]: The version not being specified helped the bug not to be detected

Comment (vigan-abd) [reply .tool-versions:1]: will this have any effect in cli runtime?

**Review by vigan-abd — APPROVED** [review]

File: .tool-versions (hunk 1 above)
Comment (tarcisiozf) [reply .tool-versions:1]: it only affects tools like ASDF or IDEs

**Pull request merged**